magma/interface.py

```diff
@@ -133,11 +133,14 @@ def _make_wires(value, wired):
     return s
 
 
+def _dict_from_decl(decl):
```

Comment:
btw, I think we can get rid of OrderedDict now because in > python3.7 builtin dict is ordered.